dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java

```diff
@@ -32,6 +32,7 @@ public class ZeppelinParameters extends AbstractParameters {
      */
     private String noteId;
     private String paragraphId;
+    private String restEndpoint;
```

Comment:
`restEndpoint` parameter should be required

Reply:
Sure, I will add a check here 
```java
    @Override
    public boolean checkParameters() {
        return StringUtils.isNotEmpty(this.noteId);
    }
```
Thx~